tests/apps/a12n/test_auth_views.py

```diff
@@ -16,11 +16,11 @@ def test_registration(
     as_anon: ApiClient,
     registration_data: RegistrationData,
     expected_user_data: UserData,
-    assert_user: Callable,
+    assert_user: GenericModelAssertion,
```

Comment:
Let's add full annotation here

```suggestion
    assert_user: GenericModelAssertion[<expected data type>],
```